www/docs/server/context.md

```diff
@@ -42,27 +42,36 @@ t2.procedure.use(({ ctx }) => { ... });
 
 ## Creating the context
 
-The `createContext()` function is called for each call to a procedure, which either comes via HTTP, a [server-side call](server-side-calls) or by using our [SSG helper](ssg-helpers):
+The `createContext()` function must be passed to whichever handler is mounting your appRouter, which may be via HTTP, a [server-side call](server-side-calls) or by using our [SSG helper](ssg-helpers).
+
+`createContext()` is called for each call to a procedure.
```

Comment:
This isn't actually true if you're using batching...we should probably clarify that it's called for "each request to tRPC" (or something like that)

Reply:
See what you make of that. I'm thinking "invocation" since server-side it may not always be accurate to refer to a 'request' since it connotes a network being involved?

Reply:
@sachinraja any further comments here?

Reply:
invocation sounds good :+1: